services/src/main/java/org/jfrog/artifactory/client/impl/jackson/RepositorySettingsMixIn.java

```diff
@@ -5,7 +5,30 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
```

Comment:
Multiple import lines have been added, since the following import line has been removed:
import org.jfrog.artifactory.client.model.repository.settings.impl.*;
I suggest we revert this change.